packages/eslint-plugin-cli/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@shopify/eslint-plugin-cli",
```

Comment:
I'd remove the `plugin` word from the package because it has nothing to do with plugins actually. As you say in the description, this contains `Shopify CLI's ESLint rules and configs`. So what about calling it `@shopify/eslint-cli` or `@shopify/cli-eslint-rules`?

Reply:
The reason I used that name is because of the already existing Shopify eslint package: https://www.npmjs.com/package/@shopify/eslint-plugin

So I just appended `cli` to that. But I think I like `@shopify/cli-eslint-rules` more 🤔 
Can we name it as we like? Should we follow the convention? (most packages are called either `eslint-plugin...` or `esling-config...` ([here](https://www.npmjs.com/search?q=eslint-plugin))

Reply:
Oh, I see. That `plugin` word may be confusing in the context of the CLI, as we have other kind of plugins... What about `cli-eslint-config`? There seem to be a lot of [packages with a similar name](https://www.npmjs.com/search?q=eslint-config) as well